core/group_setup.go

```diff
@@ -261,7 +261,7 @@ func (s *setupManager) createAndSend(keys []*key.Identity) {
 		// "DefaultResharingOffset" time from now.
 		_, transition := chain.NextRound(atLeast, s.beaconPeriod, s.oldGroup.GenesisTime)
 		group = key.NewGroup(keys, s.thr, genesis, s.beaconPeriod, s.catchupPeriod, s.scheme, s.beaconID)
-		group.TransitionTime = transition
+		group.TransitionTime = transition + int64(s.beaconPeriod.Seconds())
```

Comment:
I guess we want to add a comment here saying that it will transition 2 rounds after the minimum time the DKG takes.

Also, it might be much cleaner to just add the period above:
```
atLeast := s.clock.Now().Add(totalDKG).Add(s.beaconPeriod).Unix() 
```

WDYT?